lib/chromedriver/helper.rb

```diff
@@ -35,6 +35,20 @@ def update
       download true
     end
 
+    def compatibility
+      if File.exists?("/usr/lib64/libstdc++.so.6")
+        if `strings /usr/lib64/libstdc++.so.6 | grep -e 'GLIBC.*3.4.15'`.empty?
```

Comment:
What's the intention behind specifying `3.4.15`? What should happen if a different glibc version is detected? Is there a more robust way to check for what you want to check for? What if `strings` isn't installed? Why aren't the dots separating the version parts escaped in this regex?